samples/cloudwatch/README.md

```diff
@@ -92,25 +122,59 @@ kubectl delete pod -n amazon-cloudwatch -l app=cwagent-prometheus
+
+All of the metrics that use a common set of dimensions are grouped under a tile that is labeled with the list of dimensions. In order to find the a specific metric, you must find the tile with the corresponding set of dimensions.
+
+Because each metric has many dimensions and because many SAS Viya metrics are collected for Cloudwatch, it can
+be difficult to find a specific metric or metrics. You can use
+the reference tables in [CloudWatch SAS Viya Metrics](reference.md) to locate
+SAS Viya metrics and identify the tile with which they are associated. The tables provide these cross-reference listings: 
+
+- the metrics associated with each set of dimensions
+- the dimensions and metrics associated with each type of source (such as CAS, SAS services written in Go, or SAS services written in Java)
+- the dimensions associated with each each metric
+
+After you find the metric you are interested in, you can create a graph from
+the metric data. For example, these are the steps to create a graph of
+`go_memstats_alloc_bytes` (memory usage) of SAS Viya services that are
+written in Go:
+
+1. Go to [CloudWatch SAS Viya Metrics](reference.md) and expand the
+**By Metric** table.
+2. Locate the `go_memstats_alloc_bytes` metric. The table indicates that it is
+associated with the `'ClusterName,job,namespace,node,pod,sas_service_base,service`
+dimensions.
+3. From the AWS web console, navigate to CloudWatch.
+4. In the navigation bar on the left, choose `Metrics`.
+5. Select the `ContainerInsights/Prometheus` namespace.
+6. Select the tile with the `'ClusterName,job,namespace,node,pod,sas_service_base,service`
+dimension. A table appears that lists all of the metrics associated with that dimension.
+7. Locate the `go_memstats_alloc_bytes` metric.
+8. Click on the arrow next to the metric and select `Search for this only`.
+9. Select the checkbox for the SAS Viya services that you want to include in the
+graph.
 
 ## CloudWatch Agent Daemonset and Workload Node Placement<a name=wnp_tolerations></a>
 
 The SAS Viya workload node placement configuration uses annotations and node taints
 to schedule pods on appropriate nodes. The default CloudWatch agent is deployed
-as a daemonset and runs on every node. If you use the default CloudWatch agent, you must add tolerations to the agent's daemonset definition to enable the agent to run on nodes that are tainted for SAS Viya workload node placement.
+as a daemonset and runs on every node. If you use the default CloudWatch agent,
+you must add tolerations to the agent's daemonset definition to enable the agent
+to run on nodes that are tainted for SAS Viya workload node placement.
 
```

Comment:
Do we need to tell them where to find or how to add the code snippet to the daemonset defintion?  We've provided very detailed instructions most every else, so this section seems very terse in comparison.

Reply:
I'm not sure I understand the question...the snippet is just below the text.

Reply:
I guess I'm wondering if we need to tell them where to find the agent's daemonset definition and how to edit it.

Reply:
Added a link to the Cloudwatch doc that provides more information about the DaemonSet